gulpfile.js

```diff
@@ -150,5 +160,5 @@ function log(error) {
 }
 
 
-gulp.task('default', ['dist', 'copy']);
+gulp.task('default', ['dist', 'copy', 'deploy']);
```

Comment:
@bhomnick the `copy` task doesn't return the stream, so there's no guarantee that it will run in the order you expect. Looks like this wasn't a problem before as the original author never had `copy` as a dependency to anything, but it could cause problems with the new task.